.github/workflows/gradle.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gradle: ['7.0', '8.0']
```

Comment:
Gradle `7.0` does not support FooJay plugin, so need to bump up the minimal version to `7.6.2`. This change does NOT mean that our users need to use `7.6.2` or later.